app/javascript/styles/mastodon/components.scss

```diff
@@ -3493,6 +3495,36 @@ $ui-header-logo-wordmark-width: 99px;
   &__logo {
     margin-bottom: 12px;
   }
+
+  @media screen and (height <= 710px) {
+    &__portal {
+      display: none;
+    }
+  }
+
+  @media screen and (height <= 765px) {
+    &__portal .trends__item:nth-child(n + 3) {
+      display: none;
+    }
+  }
+
+  @media screen and (height <= 820px) {
+    &__portal .trends__item:nth-child(n + 4) {
+      display: none;
+    }
+  }
+
+  @media screen and (height <= 920px) {
+    .column-link.column-link--optional {
+      display: none;
+    }
+  }
+
+  @media screen and (height <= 1040px) {
+    .list-panel {
+      display: none;
+    }
+  }
```

Comment:
Could these be container queries instead? Not a big deal, but the available height can change depending on whether we are temporarily in single-column mode but with the advanced interface enabled, in which case a message is added to the panel. This is a nitpick though, and should not be blocking the PR.

Reply:
I won't oppose changing it to container queries in a follow-up.